runbotics-orchestrator/src/main/java/com/runbotics/domain/NotificationProcess.java

```diff
@@ -0,0 +1,110 @@
+package com.runbotics.domain;
+
+import org.mapstruct.Mapping;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "notification_process")
+public class NotificationProcess implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
+    private Long id;
+
+    @Column(name = "user_id", insertable = false, updatable = false)
+    private Long userId;
+
+    @Column(name = "process_id", insertable = false, updatable = false)
+    private Long processId;
+
+    @ManyToOne
+    private User user;
+
+    @ManyToOne
+    private Process process;
+
+    @Column(name = "type")
+    private String type;
```

Comment:
you can have more narrow types here, see system at bot entity (same case)
adjust in `NotificationBot` as well